codeclimate-wrapper

```diff
@@ -18,6 +18,29 @@ EOF
   exit 1
 }
 
+modern_docker_version() {
+  docker --version | awk '{
```

Comment:
`| grep -q "Docker version 1.11"` is functionally equivalent to this `awk` script right?